index.js

```diff
@@ -1 +1,72 @@
 // implement your API here
+
+const express = require('express');
+const db = require('./data/db');
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/api/users', (req, res) => {
+    const user = req.body;
+
+    db.insert(user)
+        .then(users => {
+            if(!user.name || !user.bio) {
```

Comment:
I would move this error handling to before we even get to the promise. . . like in between where you define user and where you make the call to the db.